ubenchmark/src/jmh/java/org/apache/calcite/benchmarks/DefaultDirectedGraphBenchmark.java

```diff
@@ -202,70 +200,14 @@ public boolean removeEdgeBenchmark(GraphState state) {
   @Benchmark
   @BenchmarkMode(Mode.AverageTime)
   @OutputTimeUnit(TimeUnit.MICROSECONDS)
-  public void removeAllVertices10Benchmark(GraphState state) {
-    state.graph.removeAllVertices(state.tenNodes);
-  }
-
-  @Benchmark
-  @BenchmarkMode(Mode.AverageTime)
-  @OutputTimeUnit(TimeUnit.MICROSECONDS)
-  public void removeAllVertices20Benchmark(GraphState state) {
-    state.graph.removeAllVertices(state.twentyNodes);
-  }
-
-  @Benchmark
-  @BenchmarkMode(Mode.AverageTime)
-  @OutputTimeUnit(TimeUnit.MICROSECONDS)
-  public void removeAllVertices30Benchmark(GraphState state) {
-    state.graph.removeAllVertices(state.thirtyNodes);
-  }
-
-  @Benchmark
-  @BenchmarkMode(Mode.AverageTime)
-  @OutputTimeUnit(TimeUnit.MICROSECONDS)
-  public void removeAllVertices40Benchmark(GraphState state) {
-    state.graph.removeAllVertices(state.fortyNodes);
-  }
-
-  @Benchmark
-  @BenchmarkMode(Mode.AverageTime)
-  @OutputTimeUnit(TimeUnit.MICROSECONDS)
-  public void removeAllVertices50Benchmark(GraphState state) {
-    state.graph.removeAllVertices(state.fiftyNodes);
-  }
-
-  @Benchmark
-  @BenchmarkMode(Mode.AverageTime)
-  @OutputTimeUnit(TimeUnit.MICROSECONDS)
-  public void removeAllVertices60Benchmark(GraphState state) {
-    state.graph.removeAllVertices(state.sixtyNodes);
-  }
-
-  @Benchmark
-  @BenchmarkMode(Mode.AverageTime)
-  @OutputTimeUnit(TimeUnit.MICROSECONDS)
-  public void removeAllVertices70Benchmark(GraphState state) {
-    state.graph.removeAllVertices(state.seventyNodes);
-  }
-
-  @Benchmark
-  @BenchmarkMode(Mode.AverageTime)
-  @OutputTimeUnit(TimeUnit.MICROSECONDS)
-  public void removeAllVertices80Benchmark(GraphState state) {
-    state.graph.removeAllVertices(state.eightyNodes);
-  }
-
-  @Benchmark
-  @BenchmarkMode(Mode.AverageTime)
-  @OutputTimeUnit(TimeUnit.MICROSECONDS)
-  public void removeAllVertices90Benchmark(GraphState state) {
-    state.graph.removeAllVertices(state.ninetyNodes);
+  public void removeVerticesBenchmark(GraphRemoveState state) {
+    state.innerState.graph.removeAllVertices(state.nodesToRemove);
   }
 
   @Benchmark
   @BenchmarkMode(Mode.AverageTime)
   @OutputTimeUnit(TimeUnit.MICROSECONDS)
-  public void removeAllVertices100Benchmark(GraphState state) {
+  public void removeAllVerticesBenchmark(GraphState state) {
```

Comment:
Just a minor note: `removeAllVerticesBenchmark ` -> `removeAllVertices` so the output is easier to read.

Reply:
Sounds good. Accepted.